modules/database/funcs.go

```diff
@@ -229,3 +232,36 @@ func (d *Database) getDBWithName(
 
 	return nil
 }
+
+//
+// Returns a *Database object with specified name and namespace
+//
+func DatabaseName(
+	ctx context.Context,
+	h *helper.Helper,
+	name string,
+) (*Database, error) {
+	// create a Database here by only filling what we need (name and namespace)
```

Comment:
you don't set the namespace as mentioned in the comment. should the comment be updated or the namespace added bellow?

Reply:
ok, 
changed it to "	// create a Database by suppplying a resource name"